src/Source/DoctrineORMQuerySourceIterator.php

```diff
@@ -45,6 +45,11 @@ class DoctrineORMQuerySourceIterator implements SourceIteratorInterface
     protected $dateTimeFormat;
 
     /**
+     * @var array
+     */
+    protected $dateFormats = array();
```

Comment:
We are using private for new things now, because it's easier to change w/o a BC-break